Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
LatestBuild is used for autotrigger scenarios, if you have linked both vsts build artifact and teamcity artifact and setup a trigger on vsts build artifact, then when a build is queued LatestBuild will be used to select the build for teamcity artifact. In short we should do the locator changes for LatestBuild also. For filtering artifact based on branches we should have default version selector but that is another feature.